week-2-practise/level-2/5-indexOf.js

```diff
@@ -5,7 +5,7 @@
 // TIP: Use the .indexOf() method
 
 function remove(arr, valueToRemove) {
-  return; // complete this statement
+  return arr.slice(arr.indexOf(valueToRemove), arr.indexOf(valueToRemove) + 1); // complete this statement
 }
 
```

Comment:
- you should use a similar solution from 4-slice to remove the item, this one returns the item
- keep the index as separate variable for readability
- don't forget to make a copy of the array using array.slice(0) so as to not modify the original array